src/hc32f460/gpio.c

```diff
@@ -1,6 +1,8 @@
 // GPIO functions on HC32F460
 //
 // Copyright (C) 2022  Steven Gotthardt <gotthardt@gmail.com>
+// Modified by @audiobrian (https://github.com/audiobrian/tld3pro_klipper)
+// and Guillaume Giraudon to add 100 pin package.
```

Comment:
The request is to not add these two lines to this file (nor to hard_pwm.c).  We don't want to track who has worked on a file in the file itself.  Use the git commit messages to track who has worked on each file.  (Adding comments on who has worked on each file rapidly leads to pages and pages of comments that make it difficult to navigate the code.)

-Kevin

Reply:
Hi Kevin.
Understood. Sorry about that.